latest_cov_build.py

```diff
@@ -13,6 +13,16 @@
     from urllib import urlretrieve
 
 
+def get_url(repos_url, download_url, is_gecko):
```

Comment:
Rename this `get_github_release_url` to make it more clear what it does.

Reply:
Also use `repo_slug` as a parameter, so you can generate `repo_url` and `download_url` dinamically.

E.g. `https://api.github.com/repos/mozilla/geckodriver/releases/latest` -> `https://api.github.com/repos/{}/releases/latest.format(repo_slug)`